cpp/tests/io/parquet_test.cpp

```diff
@@ -462,7 +464,59 @@ TYPED_TEST(ParquetWriterNumericTypeTest, SingleColumnWithNulls)
   CUDF_TEST_EXPECT_TABLES_EQUAL(expected, result.tbl->view());
 }
 
-TYPED_TEST(ParquetWriterChronoTypeTest, Chronos)
+TEST_F(ParquetWriterTest, Durations)
+{
+  std::default_random_engine generator;
+  std::uniform_int_distribution<int> distribution_d(0, 30);
+  auto sequence_d = cudf::detail::make_counting_transform_iterator(
+    0, [&](auto i) { return distribution_d(generator); });
+
+  std::uniform_int_distribution<int> distribution_s(0, 86400);
+  auto sequence_s = cudf::detail::make_counting_transform_iterator(
+    0, [&](auto i) { return distribution_s(generator); });
+
+  std::uniform_int_distribution<int> distribution(0, 86400 * 1000);
+  auto sequence = cudf::detail::make_counting_transform_iterator(
+    0, [&](auto i) { return distribution(generator); });
```

Comment:
Can we use `generate_random_table` for this?

Reply:
I guess we could. It's not a large table so there wouldn't be a lot to gain.
It's an interesting idea, but I'd leave it for a separate PR since the current code is in line with other tests. Let me know what you think.

Reply:
Sure. I'm fine with leaving it as is.